cmd/manager/main.go

```diff
@@ -115,6 +115,24 @@ func (s *server) GetSuggestions(ctx context.Context, in *api_pb.GetSuggestionsRe
 	return r, nil
 }
 
+func (s *server) ValidateSuggestionParameters(ctx context.Context, in *api_pb.ValidateSuggestionParametersRequest) (*api_pb.ValidateSuggestionParametersReply, error) {
+	if in.SuggestionAlgorithm == "" {
+		return &api_pb.ValidateSuggestionParametersReply{}, errors.New("No suggest algorithm specified")
+	}
+
+	conn, err := grpc.Dial("vizier-suggestion-"+in.SuggestionAlgorithm+":6789", grpc.WithInsecure())
+	if err != nil {
```

Comment:
It should return specific error code here (e.g. `Unavailable` ) and set message.

Reply:
I think we should delete this part: 
https://github.com/kubeflow/katib/pull/398/files/ea896ae1a4c4690ef8873ce416a019e782710f9e#diff-9e63f7b0791edcb4a53c944552b88970R119.
Since you add "random" algorithm, if it is not specified: 
https://github.com/kubeflow/katib/blob/master/pkg/controller/studyjob/katib_api_util.go#L28

Reply:
@YujiOshima By default this Error https://github.com/kubeflow/katib/pull/398/files#diff-9e63f7b0791edcb4a53c944552b88970R120 will return Unavailable code. I think, we don't need to specify this error code there. 
Right now, I handle this error here, as you said:  https://github.com/kubeflow/katib/pull/398/commits/b88030b9e9a149b335189598d70ef881f38a6bf8#diff-7a381fdaeae1295724b74c0a04f7644cR386.
Please check, if it is ok.